demo/long_video_demo.py

```diff
@@ -64,11 +62,18 @@ def parse_args():
         help='override some settings in the used config, the key-value pair '
         'in xxx=yyy format will be merged into config file. For example, '
         "'--cfg-options model.backbone.depth=18 model.backbone.with_cp=True'")
+    parser.add_argument(
+        '--font-color',
+        nargs='+',
+        type=int,
+        default=(255, 255, 255),
+        help='font color (B, G, R) of the labels in output video')
     args = parser.parse_args()
     return args
 
 
-def show_results_video(result_queue, text_info, thr, msg, frame, video_writer):
+def show_results_video(result_queue, text_info, thr, msg, font_color, frame,
+                       video_writer):
```

Comment:
```python
def show_results_video(result_queue, text_info, thr, msg, frame, video_writer, font_color=(255, 255, 255)):
```

Reply:
ah.. updated (default value is in `parse_args`, however)